packages/wrangler/src/pages/functions.tsx

```diff
@@ -0,0 +1,86 @@
+import { existsSync, readFileSync, writeFileSync } from "node:fs";
+import { FatalError } from "../errors";
+import {
+	isRoutesJSONSpec,
+	optimizeRoutesJSONSpec,
+} from "./functions/routes-transformation";
+import { pagesBetaWarning } from "./utils";
+import type { ArgumentsCamelCase, Argv } from "yargs";
+
+type OptimizeRoutesArgs = {
+	"routes-path": string;
+	"output-routes-path": string;
+};
+
+export function OptimizeRoutesOptions(yargs: Argv): Argv<OptimizeRoutesArgs> {
+	return yargs
+		.options({
+			"routes-path": {
```

Comment:
I'd make this a positional argument: `wrangler pages functions optimize-routes public/_routes.json`

Reply:
right. That was actually my first edit, but then got unsure whether that is actually a safe assumption to make, so I changed it to optional. I'll dbl check with Jacob. Thx for pointing it out!